api/leaderworkerset/v1/leaderworkerset_types.go

```diff
@@ -255,15 +257,21 @@ const (
 type RestartPolicyType string
 
 const (
-	// RecreateGroupOnPodRestart will recreate all the pods in the group if
+	// DefaultRestartPolicy will recreate all the pods in the group if
 	// 1. Any individual pod in the group is recreated; 2. Any containers/init-containers
 	// in a pod is restarted. This is to ensure all pods/containers in the group will be
 	// started in the same time.
-	RecreateGroupOnPodRestart RestartPolicyType = "RecreateGroupOnPodRestart"
+	DefaultRestartPolicy RestartPolicyType = "RecreateGroupOnPodRestart"
```

Comment:
Lets keep the parameter name as is and just add a comment that this is the default value.

Reply:
done.